README.md

```diff
@@ -11,6 +11,24 @@ $ ./gradlew ktlintApplyToIdea
 ```
 This only needs doing once to set up your IDE with the code styles.
 
+#### AWS CodeArtifact Access Set Up
+
+To access libraries stored in the AWS CodeArtifact repository an access token is required that the build script fetches
+in the background using the credentials for the `code-artifact` profile. To create this profile on your developer
+machine follow these instructions:
+
+```shell
+aws configure --profile code-artifact
+```
+
+At the prompts configure the `code-artifact` profile as follows:
+* Your AWS Access Key ID
+* Your AWS Secret Access Key
+* Default region name, `eu-west-2`
+* Default output format, `json`
+
+Note: AWS CLI must be installed on the developer workstation as a pre-requisite.
```

Comment:
Will this work with MFA access?

I think currently the EROP accounts don't use MFA on CLI access, but we aim to fix that in the medium term future

Reply:
I know that my account is MFA enabled and this approach works fine, I've not heard any developer having issues and this is the approach outlined in the[ AWS docs](https://docs.aws.amazon.com/codeartifact/latest/ug/maven-gradle.html).

So, I've looked at the [AWS CLI docs](https://docs.aws.amazon.com/cli/latest/reference/codeartifact/index.html) but haven't spotted anything specific to MFA.  

Possibly, this will have to be reviewed when the EROP accounts are updated.

Reply:
> my account is MFA enabled and this approach works fine

Do you mean that you use MFA to log into the Web Console, or when you take actions on the CLI or both?

It's a common pattern to use MFA when logging into the Web Console but to use single factor "AWS_SECRET_ACCESS_KEY" creds when working on the CLI, as it's a significant faff to create temp MFA enabled sessions each time you do the latter.

See https://repost.aws/knowledge-center/authenticate-mfa-cli

Reply:
Yes, I meant I have MFA enabled for the AWS console - not for CLI.

Reply:
That does seem like a faff :-(